libcrux-testbench/src/lib.rs

```diff
@@ -0,0 +1,4 @@
+#![no_std]
+
+pub mod mldsa;
+pub mod mlkem;
```

Comment:
Should we rename these to `mldsa65` and `mlkem768`?

Reply:
Ah, I was trying to make it more general, because in a next step I want to benchmark all the other parameter sets as well, e.g. like here https://github.com/cryspen/libcrux-iot/blob/jonas/nordic-boards/libcrux-testbench/src/mldsa.rs (maybe there is some nicer way than this feature-zoo, though).
But you couldn't have known that :sweat_smile:.

Reply:
Hm yeah. In that case I suggest we keep the names, and in each export the benchmarking functions for each variant, but I think either way is fine